pages/examples/intermediate/async-loops.mdx

```diff
@@ -0,0 +1,140 @@
+# Agent Asynchronous Loops
+
+## Introduction
```

Comment:
I think this should go under advanced - would prefer this as a guide instead of an example?

Reply:
Ok cool. Well, no idea about this. I thought about creating a guide on this but an example can be a good starting point for now. But if you prefer, i can write a guide down for this in a new PR